predicators/approaches/active_sampler_learning_approach.py

```diff
@@ -107,6 +107,7 @@ def _learn_nsrts(self, trajectories: List[LowLevelTrajectory],
 
     def _update_sampler_data(self) -> None:
         new_trajs = self._segmented_trajs[self._last_seen_segment_traj_idx:]
+        just_made_incorrect_pick = False
```

Comment:
should this be reset one level down inside the for loop?

Reply:
yes! good catch!